### PR TITLE
use new awscli endpoint configuration

### PR DESCRIPTION
**Pull request by arnauddrain**
[Aws deployed a built-in way to configure the endpoint URL](https://aws.amazon.com/fr/blogs/developer/new-improved-flexibility-when-configuring-endpoint-urls-with-the-aws-sdks-and-tools/) without installing any plugin.

### CODE REVIEW

File: pages/public_cloud/containers_orchestration/managed_kubernetes/installing-popeye/guide.de-de.md [hunk 1]
```diff
@@ -393,21 +393,20 @@ Please enter your OpenStack Password:
 Now, you need to install the CLI tools:
 
 ```
-pip install python-openstackclient awscli awscli-plugin-endpoint
+pip install python-openstackclient awscli
 ```
 
 The command install `openstack client` to generate your credentials, `aws CLI` and mandatory `endpoint plugin for aws CLI`:
```
Comment (bs-ovh) [inline pages/public_cloud/containers_orchestration/managed_kubernetes/installing-popeye/guide.de-de.md]: > and mandatory `endpoint plugin for aws CLI`:

Comment (arnauddrain) [reply pages/public_cloud/containers_orchestration/managed_kubernetes/installing-popeye/guide.de-de.md]: Good catch, I removed this and added the missing `s` to `install` 👍

**Review by Pofilo — APPROVED** [review]

**Review by lason-ovh — APPROVED** [review]
OK pour la partie S3, Swift et PCS, je laisse @antonin-a voir pour la partie Kube

**Review by antonin-a — APPROVED** [review]

**Review by OVH-Team-Guides — APPROVED** [review]

**Comment by OVH-Team-Guides** [comment]
This Pull Request will be followed by another one to updated the articles dates.
Thank you very much for your contribution @arnauddrain

**Pull request merged**